src/utils/parser/getAst.js

```diff
@@ -0,0 +1,68 @@
+import parseScriptTags from "parse-script-tags";
+import * as babylon from "babylon";
+import traverse from "babel-traverse";
+import isEmpty from "lodash/isEmpty";
+import { isDevelopment } from "devtools-config";
+
+const ASTs = new Map();
+
+function _parse(code, opts) {
+  return babylon.parse(
+    code,
+    Object.assign({}, opts, {
+      sourceType: "module",
+      plugins: ["jsx", "flow"]
+    })
+  );
+}
+
+function parse(text: string, opts?: Object) {
+  let ast;
+  if (!text) {
+    return;
+  }
+
+  try {
+    ast = _parse(text, opts);
+  } catch (error) {
+    if (isDevelopment()) {
+      console.warn("parse failed", text);
+    }
+
+    ast = {};
+  }
+
+  return ast;
+}
+
+export default function getAst(sourceText: SourceText) {
```

Comment:
Seems like `getAst` isn't the right module name/default export if you're going to standardize on using `traverseAst` to fetch the AST and handle it. Suggest making this a named export and make the module be `./ast.js`